### ISSUE TITLE
NCA output in CDISC nomenclature

### ISSUE THREAD

**Issue by roninsightrx** [humanpred/pknca#289]
hi Bill,
as mentioned a few months ago, I am interested in having the NCA output in [CDISC-compatible format](https://www.lexjansen.com/pharmasug/2017/PO/PharmaSUG-2017-PO01.pdf) (and input as well, but that can be tackled later). I'd be happy to work on a feature in PKNCA or a subpackage that handles this. My initial questions would be:
- would you be OK having this as a feature in PKNCA, or would you prefer to have it as a subpackage (I saw your `PKNCA.portation` stub from a few years ago). I actually don't think it's that much code that would be required to allow CDISC output, just a function for remapping of variable names, so I think my suggestion would be to include in PKNCA.
- could you give any high-level suggestions on how you would prefer to architect this? E.g. would it be a `cdisc=TRUE/FALSE` argument to `pk.nca()`? Or only as a separate user-facing function?
Please LMK any thoughts.

**Comment by billdenney** [comment]
Yes, I'm very interested in this.  Thank you for interest in contributing!

I think that the best place to make the main change would be the `as.data.frame.PKMCAresults()` adding a new `out_format = "SDTM"`.

The other set of changes that would need to be made is in the `add.nca.parameter()` function to give SDTM PPTEST and PPTESTCD columns.

I would consider a major, breaking change as part of this to make the following changes that revise the main column structure of the internal data.frame so that it no  longer uses inaccurate PPTESTCD values internally, perhaps changing to PKNCACD or similar.  When I first wrote PKNCA, my understanding of SDTM was less complete, so I didn't realize that there were controlled terms.  And, SDTM PPTESTCD names are not fully precise, so the names need to be maintained.

The last change I'm suggesting should probably be the first made since it can probably be a simple search and replace at this point.

**Comment by billdenney** [comment]
Closed in #403 and #536
